service/telegram/telegram.go

```diff
@@ -33,8 +35,12 @@ func New(apiToken string) (*Telegram, error) {
 
 // AddReceivers takes Telegram chat IDs and adds them to the internal chat ID list. The Send method will send
 // a given message to all those chats.
-func (t *Telegram) AddReceivers(chatIDs ...int64) {
-	t.chatIDs = append(t.chatIDs, chatIDs...)
+func (t *Telegram) AddReceivers(chatIDs ...string) {
+	for _, v := range chatIDs {
+		if chatID, err := strconv.ParseInt(v, 10, 64); err != nil {
```

Comment:
Please change the format of err check to:

```go
chatID, err := strconv.ParseInt(v, 10, 64)
if err != nil {
...
}
```

This is only to maintain a consistent code style.